File: imagepicker/src/main/java/com/gipl/imagepicker/listener/IImageListResult.java
package com.gipl.imagepicker.listener;

import com.gipl.imagepicker.models.ImageResult;

import java.util.ArrayList;

public interface IImageListResult {

    /**
     * for muti select you will receive result from this listener
     *
     * @param imageResults :array of ImageResult(image path and bitmap)
     */
    void onReceiveImageList(ArrayList<ImageResult> imageResults);
}

File: imagepicker/src/main/java/com/gipl/imagepicker/listener/IImageListResult.kt
package com.gipl.imagepicker.listener

import com.gipl.imagepicker.models.ImageResult
import com.gipl.imagepicker.exceptions.ImageErrors
import java.util.ArrayList

interface IImageListResult {
    /**
     * for muti select you will receive result from this listener
     *
     * @param imageResults :array of ImageResult(image path and bitmap)
     */
    fun onReceiveImageList(imageResults: ArrayList<ImageResult?>?)
}